chain/x/shard/types/genesis.go

```diff
@@ -5,10 +5,5 @@ func DefaultGenesis() *GenesisState {
 }
 
 func (g *GenesisState) Validate() error {
```

Comment:
Is this meant to still exist?

Reply:
forgot to add that [feat: add genesis validation](https://github.com/Argus-Labs/world-engine/pull/169/commits/2132db263d68ef9b54e4218620f28422e4710076)